### ISSUE TITLE
i trained my custom nanodet model and convert t onnx to tflite

### ISSUE THREAD

**Issue by arunmack789** [RangiLyu/nanodet#421]
using tflite file from onnx i get infernce arrays with values like thus
[ 3.2581491e-03  7.2175977e-03 -3.7810624e+00 -3.5439680e+00
 -3.0214891e+00 -3.5125756e+00 -4.5226998e+00 -4.8461933e+00
 -4.3985019e+00 -3.8702958e+00 -2.6041222e+00 -2.9508047e+00
 -3.8784418e+00 -4.4361591e+00 -4.3928189e+00 -4.1803117e+00
 -4.6547728e+00 -5.2755866e+00 -3.5790832e+00 -3.5240192e+00
 -3.5083601e+00 -3.7697649e+00 -4.0104628e+00 -4.1850638e+00
 -4.3945179e+00 -4.4849253e+00 -3.0874686e+00 -3.6977808e+00
 -3.9248106e+00 -4.0122795e+00 -4.0581307e+00 -4.1862755e+00
 -3.8854568e+00 -3.5769753e+00]

**Comment by ubergeekNZ** [comment]
I am getting the same thing with using opencv cv2.dnn.readNetFromONNX using converted custom model to onnx. I wondering if it is missing something else. In the nccn demo, it seems to include https://github.com/RangiLyu/nanodet/blob/main/demo_ncnn/nanodet.cpp. Is there something missing in onnx or tflite?

**Comment by ubergeekNZ** [comment]
I got it working with onnx

**Comment by Bluessea** [comment]
I have the same problem. 
The result score of onnx reasoning is very low. And torch run reasoning is fine.
I don't know why.

**Comment by mathTaoTao** [comment]
> 

I have the same problem as you,I use shuffleNetV2 as backbone, the result score of onnx is very low,have you solved this problem?Thanks.

**Comment by mathTaoTao** [comment]
>

I alse use onnxruntime and I use shuffleNetV2 as backbone, the result score of onnx is very low and differ from the result of pytorch
